content/HelloSign - Alejandro Alvarez Diego Barral.md

```diff
@@ -0,0 +1,93 @@
+	-**client_id**: client_id which can be found in the HelloSign web app
+	-**signers**: array of users that will ultimately sign the document
+	-**signing_redirect_url**: this url is where the user will be redirected after he signed the document
+	-**custom_fields**: since the document may be loaded with different information for different users, here is where this information should be passed to HelloSign, so a new    document with the corresponding information is rendered. An object should be passed here, with the keys named exactly as it was set up in the template.
+
+Next you can see an example of a call to create a new document:
+
+<img src="images/hellosign-2.png" alt="Create signature request" />
+
+So with this last bit of code we created an embedded signature request. In the response from the API we can find some useful information, for instance, the `sign_url`. You need to send this to the frontend of your application so the document can be embedded.
+
+And that's it! After the frontend of your application renders the document and the user signs, he'll be redirected to the callback url and will continue with the flow. 
+
+But what if you want to validate if the document was in fact signed? Easy, HelloSign webhooks.
+
+### Webhooks
+
+HelloSign provides webhooks which you can configure to hit an endpoint in your API, and you'll start receiving different events. You can find thorough documentation regarding webhooks and events [here](https://app.hellosign.com/api/eventsAndCallbacksWalkthrough). You just need to listen for the events you need, and do stuff accordingly.
+
+## Frontend
+
+### HelloSign Embedded
+
+HelloSign provides a [Javascript library for React](https://github.com/hellosign/hellosign-embedded) that allows embedding the documents to sign in your own application with minimal effort. In their repo, linked above, you can find the steps to install it. 
+
+## Usage
+
+To begin with, we must instantiate the HelloSign client. 
+
+<img src="images/hellosign-3.png" alt="Instantiate HelloSign web client" />
+
+Next thing we need, is get the urls of the documents to sign. This should be provided by the backend, as we mentioned above. Once we get this urls, we can show them. In order to display a document, we need to have a `<div>` with a specific id. 
+
+<img src="images/hellosign-4.png" alt="HelloSign document container" />
+
+Then, the document can be displayed in said container. This is the way to do that:
+
+<img src="images/hellosign-5.png" alt="Show HelloSign document" />
+
+You are able also to handle the clients events, such as `sign` for instance.
+
+<img src="images/hellosign-6.png" alt="Handle HelloSign events" />
+
+You can find more about events to handle [here](https://github.com/hellosign/hellosign-embedded/wiki/API-Documentation-(v2)#events)
+
+## Summary
+
+HelloSign and the tools that they provide, its an easy way to allow the users of your application to legally sign documents, in a pretty straight forward way. 
```

Comment:
This summary is to be included at the end of the article? I think that instead of a summary you can add a conclusion on how it has helped you work on a particular problem.